gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/GateConfig.groovy

```diff
@@ -83,6 +84,12 @@ class GateConfig {
   }
 
   @Bean
+  @ConditionalOnProperty("redis.configuration.secure")
```

Comment:
if the redis instance is locked down such that you can't issue `config set` ops against it, set `redis.configuration.secure=true` and configure the redis instance directly (this is needed for AWS elasticache redis)